src/DependencyInjection/Configuration.php

```diff
@@ -37,6 +37,7 @@ public function getConfigTreeBuilder()
                     ->prototype('scalar')->end()
                     ->defaultValue(['main'])
                 ->end()
+                ->scalarNode('user_class')->cannotBeEmpty()->end()
```

Comment:
Default to an/the entity in this bundle?

Reply:
Can't do this as the user class is a superclass which is not an entity of itself.